src/main/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/rca/persistence/Persistable.java

```diff
@@ -54,4 +56,10 @@
   <T extends ResourceFlowUnit> void write(Node<?> node, T flowUnit) throws SQLException, IOException;
 
   void close() throws SQLException;
+
+  List<Result<Record>> getRecordsForAllTables();
```

Comment:
Result and Record are closely coupled with SQL. Do we want to expose that level of details in Persistable interface ?

Reply:
Making it so that we can access the RCAs only via the Rest API.